### DISCUSSION TITLE
Unable to figure out data directory.

### DISCUSSION THREAD

**Discussion by ppebb** [mfussenegger/nvim-jdtls#253] [Q&A] [answered]
I've tried replicating the example in the config, using pwd to get the project name, but I just can't get it to work.

Where is the data directory supposed to point to in relation to the folder that contains the .git and build.gradle files. As of now I've got it pointing to where my .git and my gradle files are, it creates some jdtls project folder and a .metadata folder but then in neovim it still says the file is not part of a project and only syntax errors will be shown.

**Comment by mfussenegger** [comment, marked as answer]
Last time I checked using the project directory as data directory led to issues. I'd recommend using a dedicated data directory. Where exactly on the filesystem you place it doesn't matter as long as you've write and read permissions on it.

**Reply by ppebb** [reply]
This works now. Tried doing the same thing before and it wasn't working, weird.
